tests/lts_compatibility.py

```diff
@@ -504,20 +503,18 @@ def run_ledger_compatibility_since_first(args, local_branch, use_snapshot):
                     )
 
                 # We accept ledger chunk file differences during upgrades
-                # from 2.x-rc7 to 2.x-rc8 and from 1.x to 2.x-rc8. This
-                # is necessary because the ledger files may not be chunked
-                # at the same interval between those versions (see
-                # https://github.com/microsoft/ccf/issues/3613; 1.x ledgers
-                # do not contain the header flags to synchronize ledger chunks).
+                # from 1.x to 2.x post rc7 ledger. This is necessary because
+                # the ledger files may not be chunked at the same interval
+                # between those versions (see https://github.com/microsoft/ccf/issues/3613;
+                # 1.x ledgers do not contain the header flags to synchronize ledger chunks).
                 # This can go once 2.0 is released.
                 current_version_past_2_rc7 = primary.version_after("ccf-2.0.0-rc7")
+                has_pre_2_rc7_ledger = (
+                    not current_version_past_2_rc7 or has_pre_2_rc7_ledger
+                )
                 is_ledger_chunk_breaking = (
-                    not previous_major_version
-                    or previous_major_version == 1
-                    or not previous_version_past_2_rc7
-                ) and current_version_past_2_rc7
-                previous_major_version = primary.major_version
-                previous_version_past_2_rc7 = current_version_past_2_rc7
+                    has_pre_2_rc7_ledger and current_version_past_2_rc7
+                )
```

Comment:
There might be a problem when the version is equal to RC7, but I think that won't be triggered by any of the CI runs (and we don't care once 2.0 is released).